README.md

```diff
@@ -3,40 +3,48 @@
 [![aws-lc-fips-sys](https://img.shields.io/badge/aws--lc--fips--sys-crates.io-important)](https://crates.io/crates/aws-lc-fips-sys)
 
 # AWS Libcrypto for Rust
+
```

Comment:
This readme is technically generated by a combination of the aws-lc-rs lib.rs documentation and the contents of https://github.com/aws/aws-lc-rs/blob/main/aws-lc-rs/README.tpl

Did you regenerate this using the [`cargo readme`](https://crates.io/crates/cargo-readme)

Reply:
Ahh.. thanks for the reminder!  I'll refactor this README change.